templates/scripts/run_dm-master_scale.sh.tpl

```diff
@@ -17,9 +17,9 @@ cd "${DEPLOY_DIR}" || exit 1
 {{- end}}
 
 {{- if .NumaNode}}
-exec numactl --cpunodebind={{.NumaNode}} --membind={{.NumaNode}} bin/bin/dm-master \
+exec numactl --cpunodebind={{.NumaNode}} --membind={{.NumaNode}} bin/dm-master/dm-master \
```

Comment:
The directory layout seems weird, why not `bin/dm/dm-master` or `bin/dm/dm-worker`? Or place the `dm-master/worker` in `bin` directly?

Reply:
same issue as https://github.com/pingcap/tiup/pull/554#discussion_r448338005
I think it need some refactor to it, or it's restricted by the tar bar format.

there are existing such weird in cluster:
```
➜  tiup git:(dm_test) ✗ ag bin/bin ./templates
templates/scripts/run_grafana.sh.tpl
25:exec numactl --cpunodebind={{.NumaNode}} --membind={{.NumaNode}} bin/bin/grafana-server \
27:exec bin/bin/grafana-server \
```

Reply:
I think the tarball layout should be changed to make the final layout more natural.